data/test/scenarios/unknown_scenario_warning.cfg

```diff
@@ -38,7 +46,12 @@
     [/event]
 )}
 
-# Not a branching scenario, but the only route uses a variable
+#####
+# API(s) being tested: [scenario]next_scenario
+##
+# Expected end state:
+# The endlevel for $chosen_branch doesn't print a warning (it not existing is accepted).
```

Comment:
The variable being undefined is accepted. This is to avoid a false positive triggering the warning, with the assumption being that something will set that value while the scenario is running.